deploy/docs/Non_Helm_Installation.md

```diff
@@ -0,0 +1,493 @@
+
+Download Helm to generate the yaml files necessary to deploy by running
+
+```bash
+brew install kubernetes-helm
+```
+
+Download the Prometheus Operator `prometheus-overrides.yaml` by running
+
+```bash
+$ cd /path/to/helm/charts/  
+$ curl -LJO https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/v0.9.0/deploy/helm/prometheus-overrides.yaml
+```
+
+Before installing `prometheus-operator`, edit `prometheus-overrides.yaml` to define a unique cluster identifier. The default value of the `cluster` field in the `externalLabels` section of `prometheus-overrides.yaml` is `kubernetes`. If you will be deploying the metric collection solution on multiple Kubernetes clusters, you will want to use a unique identifier for each. For example, you might use “Dev”, “Prod”, and so on.
+
+__NOTE__ It’s fine to change the value of the `cluster` field, but don’t change the field name (key).
+
+__NOTE__ If you plan to install Prometheus in a different namespace than you deployed Fluentd to in Step 1, or you have an existing Prometheus you plan to apply our configuration to running in a different namespace,  please update the remote write API configuration to use the full service URL like, `http://collection-sumologic.sumologic.svc.cluster.local:9888`.
+
+You can also [Filter metrics](#filter-metrics) and [Trim and relabel metrics](#trim-and-relabel-metrics) in `prometheus-overrides.yaml`.
+
+Install `prometheus-operator` by generating the yaml files using Helm:
+
+```bash
+$ helm template stable/prometheus-operator --name prometheus-operator --set dryRun=true -f prometheus-overrides.yaml > prometheus.yaml
+$ kubectl apply -f prometheus.yaml
+```
+
+Verify `prometheus-operator` is running:
+
+```sh
+kubectl -n sumologic logs prometheus-prometheus-operator-prometheus-0 prometheus -f
+```
+
+At this point setup is complete and metrics data is being sent to Sumo Logic.
+
+### Missing metrics for `controller-manager` or `scheduler`
+
+Since there is a backward compatibility issue in the current version of chart, you may need to follow a workaround for sending these metrics under `controller-manager` or `scheduler`:
+
+```sh
+kubectl -n kube-system patch service prometheus-operator-kube-controller-manager -p '{"spec":{"selector":{"k8s-app": "kube-controller-manager"}}}'
+kubectl -n kube-system patch service prometheus-operator-kube-scheduler -p '{"spec":{"selector":{"k8s-app": "kube-scheduler"}}}'
+kubectl -n kube-system patch service prometheus-operator-kube-controller-manager --type=json -p='[{"op": "remove", "path": "/spec/selector/component"}]'
+kubectl -n kube-system patch service prometheus-operator-kube-scheduler --type=json -p='[{"op": "remove", "path": "/spec/selector/component"}]'
+```
+
+### Additional configuration options
+
+### Metrics
+
+#### Filter metrics
```

Comment:
This section is a general section and would need to be duplicated in 3 different places, this should be a seperate doc

Reply:
Agreed, but let's save that for another PR to unblock merging all of these.